drizzlepac/haputils/pipeline_graphics.py

```diff
@@ -467,3 +471,27 @@ def build_astrometry_plots(pandas_file,
     
     return resids_plot_names
 
+if __name__ == "__main__":
+    """Simple command-line interface. That is all.
+    """
+    # process command-line inputs with argparse
+    parser = argparse.ArgumentParser(description='Generate pipeline astrometry quality assessment plots based'
+                                                 ' on information stored in the user-specified .h5 file.')
+    parser.add_argument('input_filename',
+                        help='Name of the input .h5 file produced by diagnostic_json_harvester.py that holds '
+                             'the information to plot.')
+
+    parser.add_argument('-d', '--output_dir', required=False, default=None,
```

Comment:
The short option here should be '-o' or perhaps '-p' (for '--output_path' as noted in the help string) instead, as '-d' is already used by other related code (like runsinglehap) for diagnostic mode.

Reply:
Implemented @stsci-hack's recommended changes in commit [45ed123](https://github.com/spacetelescope/drizzlepac/pull/768/commits/45ed12314aa3a739e78dbff367351982f7e4b550).